<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceClass</class>
 <widget class="QMainWindow" name="InterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CHAI3D</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/chai3d.ico</normaloff>:/chai3d.ico</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="Settings" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupOptions">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Viewing options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="showFrame">
            <property name="text">
             <string>Show frame</string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showBoundaryBox">
            <property name="text">
             <string>Show boundary box</string>
            </property>
            <property name="shortcut">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showTriangles">
            <property name="text">
             <string>Show triangles</string>
            </property>
            <property name="shortcut">
             <string>T</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showWireframe">
            <property name="text">
             <string>Show wireframe</string>
            </property>
            <property name="shortcut">
             <string>W</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showEdges">
            <property name="text">
             <string>Show edges</string>
            </property>
            <property name="shortcut">
             <string>E</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showNormals">
            <property name="text">
             <string>Show normals</string>
            </property>
            <property name="shortcut">
             <string>N</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupModels">
         <property name="title">
          <string>Model list</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListView" name="folderList"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Application.qrc"/>
 </resources>
 <connections/>
</ui>
